Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -428,8 +428,26 @@ export default function NewMeterWizard(props: {}) {
     function disableNext() { return error.length > 0 };
 
     return (
-        <div style={{padding: 10, height: 'inherit', overflowY: 'hidden'}}>
-            <h2>New Meter Wizard</h2>
+        <div style={{ padding: 10, height: 'inherit', overflowY: 'hidden' }}>
+            <h2 style={{ width: 300, display: 'inline-block' }}>New Meter Wizard</h2>
+            <div style={{ height: 40, width: 'calc(100% - 300px)', display: 'inline-block' }}>
+                <ProgressBar width={'100%'} height={40}
+                    steps={[
+                        { id: generalStep, long: 'Meter Information', short: 'Meter' },
+                        { id: locationStep, long: 'Substation Information', short: 'Location' },
+                        { id: eventChannelsStep, long: 'Event Channel Configuration', short: 'Event' },
+                        { id: trendChannelsStep, long: 'Trend Channel Configuration', short: 'Trend' },
+                        { id: assetStep, long: 'Asset Configuration', short: 'Asset' },
+                        { id: connectionStep, long: 'Network Model', short: 'Network*' },
```

Comment:
This could be confusing as opposed to using the `Asset Connection` label used elsewhere. If we want to use `Network Model`, it should probably be changed in other places in the application.

Reply:
```suggestion
                        { id: connectionStep, long: 'Asset Connection', short: 'Connection*' },
```